src/main/car/functionality/DealershipUI

```diff
@@ -0,0 +1,141 @@
+	private static void makeDealerGUI(DealerGroup dealers) {
+	    //dealer information screen
+		JPanel dealerDetails = new JPanel();
+		dealerDetails.setLayout(new GridLayout(3,1,0,10));
+		JPanel Options = new JPanel();
+		Options.setLayout(new GridLayout(5,3,10,10));
+		JPanel specificVehicle = new JPanel();
+		specificVehicle.setLayout(new GridLayout());
+		JLabel ID = new JLabel("Dealer: ID here");
+		JLabel Name = new JLabel("Name: ");
+		JLabel Renting = new JLabel("Renting: ");
+		JLabel Acquisition = new JLabel("Vehicle Acquisition: ");
+		JLabel Transfer = new JLabel("Transfer Inventory to: ");
+		JLabel Export = new JLabel("Export: ");
+		JTextField enterName = new JTextField("Dealer Name here");
+		JTextField transferInventory = new JTextField("Receiving Dealer ID here");
+		//change name of dealer
+		JButton changeName = new JButton("save");
+		changeName.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				searchedDealer.setDealerName(enterName.getText());
+			}
+		});
+		//adjust rent boolean value of dealer
+		JButton disableRent = new JButton("disable");
+		disableRent.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				searchedDealer.disableRenting();
+			}
+		});
+		JButton enableRent = new JButton("enable");
+		enableRent.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				searchedDealer.enableRenting();
+			}
+		});
+		//adjust vehicle acquisition boolean value of dealer
+		JButton disableAcquisition = new JButton("disable");
+		disableAcquisition.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				searchedDealer.enableDealerVehicleAcquisition();
+			}
+		});
+		JButton enableAcquisition = new JButton("enable");
+		enableAcquisition.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				searchedDealer.disableDealerVehicleAcquisition();
+			}
+		});
+		//checks inventory for rented cars first before transferring entire inventory
+		JButton transferAllVehicles = new JButton("transfer");
+		transferAllVehicles.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				for(Map.Entry<String, Vehicle> vehicle : searchedDealer.getVehicleInventory().entrySet()) {
+					if(searchedDealer.checkIfCarIsRented(vehicle.getValue())) {
+						//transfer did not occur because a vehicle in the dealer's inventory is rented
+						return;
+					}
+				}
+				dealers.transferInventory(searchedDealer.getDealerID(), transferInventory.getText());
+			}
+		});
+		
+		//export function that still needs work
+		JButton XMLExport = new JButton("XML");
+		XMLExport.setVisible(false);
+		//Ask about how to export this to JSON
+		JButton JSONExport = new JButton("JSON");
+		JSONExport.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				searchedDealer.toString();
```

Comment:
XML export is not necessary! :) Also you should be able to call VehicleJSONParser.write(searchedDealer) for the JSON export

Reply:
Gotcha, tested removing the button altogether rather than making it invisible to see how it impacted the GUI, and adjusted the Options panel to make the Specific Vehicle button a normal size. Also added the suggested method to the JSON button.